src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/GoalEntityMapperUpdateEntityFromDomainTest.kt

```diff
@@ -0,0 +1,495 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.mapper
```

Comment:
This is a new test class
Its massive! which is one of the reasons I've written it as a new test class rather than updating the existing one
All the tests in this class are about testing the one new method in `GoalEntityMapper` - ie. the new `updateSteps` method